.NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs

```diff
@@ -214,12 +215,14 @@ public void TestTimeParseLuis()
         [TestMethod]
         public void TestTimeParseMealTimeLuis()
         {
-            BasicTest("I'll be back 12 o'clock lunchtime", "T12");
-            BasicTest("I'll be back 12 o'clock lunch", "T12");
-            BasicTest("I'll be back 8 o'clock dinner", "T20");
-            BasicTest("I'll be back 8 o'clock breakfast", "T08");
-            BasicTest("I'll be back at lunch 12 o'clock", "T12");
+            //We only support lunchtime now
+            //BasicTest("I'll be back 12 o'clock lunch", "T12");
```

Comment:
We'll add a task to potentially bring some of these back only as disambiguation.

Reply:
done